docs: changed p to p tag for more undertanding

ajitsinghkaler opened the pull request:
Earlier p tag was only defined by now written tag with p so that it is more clear

Fixes #34922

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: 34922


## What is the new behavior?
Added p tag

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->


## Other information

ajitsinghkaler commented:
@kapunahelewong can you please review this

ericmartinezr commented on aio/content/guide/attribute-directives.md:
```diff
@@ -173,7 +173,7 @@ Here's the updated directive in full:
 
 Run the app and confirm that the background color appears when
-the mouse hovers over the `p` and disappears as it moves out.
+the mouse hovers over the `p` tag and disappears as it moves out.
```
How about 
```
`<p>` tag
```

ajitsinghkaler commented:
@gkalpak can you please review this too

aikidave commented on aio/content/guide/attribute-directives.md:
```diff
@@ -173,7 +173,7 @@ Here's the updated directive in full:
 
 Run the app and confirm that the background color appears when
-the mouse hovers over the `p` and disappears as it moves out.
+the mouse hovers over the `<p>` tag and disappears as it moves out.
```
I would recommend the following:

"...the pointer hovers over the paragraph element and disappears as the pointer moves out."

Why this recommendation:

- The term, "pointer," is generally a better term to use than "mouse" (for a variety of reasons, but localization is probably one of the bigger ones.
- The phrase "paragraph element" is a lot clearer
- Using words like "it" and "this" can be challenging. (Some studies show that users get easily confused by what "it" or "this" refers to.) Repeating the phrase, "the pointer" adds greater clarity.

Hope this helps!

--Your friendly neighborhood tech writer

ajitsinghkaler replied:
Thanks for such nice recommendations @aikidave. You make great points I'll make these changes.

aikidave replied:
Awesome! And I'm sorry that my comment posted twice. I am not sure why I didn't see it before. :)

ajitsinghkaler replied:
No Problem you already helped me a lot with those awesome comments

ajitsinghkaler committed:
docs: change p to pointer for better understanding

earlier p sigly was used to define a paragraph element which was not clear what we want to refer changed it to paragrapg element for better understanding

Fixes #34922

aikithoughts reviewed (approved):
Looks good to me!

kapunahelewong commented:
Thank you for this, @ajitsinghkaler. Could you try rebasing to see if it will fix those failures?

ajitsinghkaler commented:
@kapunahelewong All checks have passed can you please add it to merge label to it.
